### PR TITLE
Change RPM Setting

### PR DESCRIPTION
**Pull request by PraneelSeth**
Created a new branch that is a repository of 6901 main. This one is up to date with main and rebased/merged and everything with new shooter RPM management code.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
   public static final class ShooterConstants {
     public static final int kLeftShooterMotorPort = 1;
     public static final int kRightShooterMotorPort = 2;
-    public static final int kShooterTestRPM = 1950;
```
Comment (Piphi5) [inline src/main/java/frc/robot/Constants.java:19]: Keep a variable like kShooterFenderRPM = 1950

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -70,7 +69,7 @@ private void configureButtonBindings() {
     new JoystickButton(m_operatorController, Button.kX.value)
         .whenPressed(
             () -> {
-              m_shooter.setRPM(ShooterConstants.kShooterTestRPM);
+              m_shooter.setRPM(500);
```
Comment (Piphi5) [inline src/main/java/frc/robot/RobotContainer.java:72]: Use the variable above here instead

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 3]
```diff
@@ -16,35 +16,45 @@
 
 public class Shooter extends SubsystemBase {
   // Spark Max Motor Controller Object
-  CANSparkMax m_motorLeader =
+  CANSparkMax m_shooterMotorLeader =
       new CANSparkMax(ShooterConstants.kLeftShooterMotorPort, MotorType.kBrushless);
-  CANSparkMax m_motorFollower =
+  CANSparkMax m_shooterMotorFollower =
       new CANSparkMax(ShooterConstants.kRightShooterMotorPort, MotorType.kBrushless);
```
Comment (Piphi5) [inline src/main/java/frc/robot/subsystems/Shooter.java:22]: Change the port constants to leader follower instead of left and right

**Review by Piphi5 — CHANGES_REQUESTED** [review]

**Comment by PraneelSeth** [comment]
I made the changes that you requested and everything is up to date with the frc6901 upstream

**Review by Piphi5 — APPROVED** [review]

**Pull request merged**